content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,227 @@
+=================
+PDF quote builder
+=================
+
+The *PDF Quote Builder* in Odoo *Sales* provides the opportunity to export a fully-custom PDF file
+for quotes, showcasing various information and design elements, instead of just the price and total.
+
+The PDF Quote Builder can be used to add header pages, product descriptions, and footer pages to a
+quote.
+
+Having a customized PDF in quotes provides a heightened conclusion to the shopping experience for
+customers, and adds an elegant level of professionalism to a company.
+
+Configuration
+=============
+
+In order to add custom PDF files for quotes, the :guilabel:`PDF Quote builder` feature *must* be
+configured.
+
+To do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`. Then, on the
+:guilabel:`Settings` page, scroll to the :guilabel:`Quotations & Orders` section, and locate the
+:guilabel:`PDF Quote builder` feature.
+
+.. image:: pdf_quote_builder/pdf-quote-builder-feature.png
+   :align: center
+   :alt: The PDF Quote Builder feature located on the Settings page of the Sales application.
+
+Here, custom :guilabel:`Header pages` and :guilabel:`Footer pages` can be uploaded. To upload
+either, click the :guilabel:`Upload your file` button, or the :guilabel:`✏️ (pencil)` icon to the
+right of the desired field, and proceed to locate, select, and upload the desired PDF file.
```

Comment:
Header en footer could also directly be added in a quotation template, so you could have different ones on a/some template(s) ^^

Reply:
Got it. Just added a note admonition to include this. Thanks for the suggestion. 👍